test/quantization/core/experimental/test_quantized_tensor.py

```diff
@@ -10,8 +10,8 @@
 
 class TestQuantizedTensor(unittest.TestCase):
     r""" Tests quantize_APoT result (int representation) on random 1-dim tensor
-        and hardcoded values for b, k by comparing to uniform observer
-        (non-uniform observer reduces to uniform for k = 1)
+        and hardcoded values for b, k by comparing to uniform quantization
+        (non-uniform quantization reduces to uniform for k = 1)
```

Comment:
seems this was changed in the wrong PR, should be done in the previous PR? same goes for some of the other changes in this PR